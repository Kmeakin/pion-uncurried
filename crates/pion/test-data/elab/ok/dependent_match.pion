let T = fn(b) -> match b {
    false => Type, 
    true => Bool
};

let f : T
= fn(x: Bool) => match x {
    false => Type,
    true => false,
};