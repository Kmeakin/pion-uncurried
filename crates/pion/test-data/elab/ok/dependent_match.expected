let T : Type = fn(b : Bool) -> match b {
    false => Type,
    true => Bool,
};

let f : fn(b : Bool) -> match b {
    false => Type,
    true => Bool,
} = fn(x : Bool) => match x {
    false => Type,
    true => false,
};
